ufo/ufo-resources.c

```diff
@@ -604,6 +604,8 @@ add_program_from_source (UfoResourcesPrivate *priv,
         build_options = g_string_new (priv->build_opts->str);
         opt_append_device_options (build_options, priv, i);
         opt_append_include_paths (build_options, priv);
+        g_string_append (build_options, " ");
+        g_string_append (build_options, options);
```

Comment:
How come nobody ever noticed this?

Reply:
I redid this part a month ago, that slipped through :-( By the way, you should check if options is not `NULL` and only then add it.

Reply:
Right, coming up.